examples/somersaultecu.py

```diff
@@ -1518,6 +1523,19 @@ class SomersaultSID(IntEnum):
                     bit_position=None,
                     sdgs=[],
                 ),
+                ValueParameter(
+                    oid=None,
+                    short_name="bribe",
```

Comment:
Why the extra param?

Reply:
because I wanted a state chart which has a pre condition state ref that depends on the value of a parameter. Doing it this way seemed to be a somewhat intuitive scenario to me...